### PR TITLE
MAKE MY PORTFOLIO RESPONSIVE ON LARGER SCREENS

### PR DESCRIPTION
**Pull request by carlylechia**
On this branch,
 - I used media queries to make my portfolio site responsive to desktop screens with widths higher than 992px.
 - Designed my project to look as close to Figma template 2 as possible.
 - Added a few transitions and visual effects with CSS.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1040,3 +1040,2010 @@ body {
     color: black;
   }
 }
+
+@media screen and (min-width: 620px) {
```
Comment (Hamzaoutdoors) [inline style.css:1044]: - [ ] As you can clearly see in the screenshot below you have another scrollbar added in your Portfolio the thing that deforms your project design, Kindly work on refactoring that by adding `overflow: hidden;` in your CSS, you can read more about that [here](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow)


overflow

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @carlylechia, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README

## Required Changes ♻️

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- As this project scales, you might find it handy to avoid repeating CSS rules by:
- grouping elements that have the same style e.g:
```
.middle-section,
.later-section {
  font-family: Aria, sans-serif;
  font-style: normal;
}
```
- or using a class name that can be re-used e.g:
```
.d-flex {
  display: flex;
  align-items: center;
}

HTML:

<div class="section-one d-flex"></div>

<div class="section-two d-flex"></div>
```
You can check redundancy [here ](https://courses.cs.washington.edu/courses/cse154/20su/resources/assets/css-redundancy-checker/index.html)

Your redundancies : 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (carlylechia) [reply style.css:1044]: Thank you

Comment (carlylechia) [reply style.css:1044]: > * [ ]  As you can clearly see in the screenshot below you have another scrollbar added in your Portfolio the thing that deforms your project design, Kindly work on refactoring that by adding `overflow: hidden;` in your CSS, you can read more about that [here](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow)
> 
> overflow

Hello Hamza, about this, I already used "overflow: hidden" right at the beginning and it was supposed to run through the entire project.

Comment (Hamzaoutdoors) [reply style.css:1044]: Hey @carlylechia try to use `overflow-y: hidden` or `overflow-x: hidden`

**Pull request merged**